src/test/java/com/epam/training/array/ArraySorterTest.java

```diff
@@ -1,13 +1,67 @@
 package com.epam.training.array;
 
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
 public class ArraySorterTest {
 
-    //TODO: uncomment test when junit dependency is added
-    //@Test
+    @Test
     public void testSortShouldSortWhenPositiveNumbers(){
+    	//given(pre-conditions)
+    	int[] firstArray = new int[] {8, 43, 27, 68, 112, 7612, 18, 320, 224, 27, 130};
+    	ArraySorter arraySorter = new ArraySorter();
+    	Array arrayForSort = new Array(firstArray);
+    	//when
+    	Array resultArray = arraySorter.sort(arrayForSort);
+    	int[] expectedArray = {8, 18, 27, 27, 43, 68, 112, 130, 224, 320, 7612};
+    	Assert.assertArrayEquals(expectedArray, resultArray.getItems());
 
     }
+    
+    @Test
+    public void testSortShouldSortWhenNegativeNumbers(){
+    	//given(pre-conditions)
+    	int[] firstArray = new int[] {-8, -43, -27, -68, -112, -7612, -18, -320, -224, -27, -130};
+    	ArraySorter arraySorter = new ArraySorter();
+    	Array arrayForSort = new Array(firstArray);
+    	//when
+    	Array resultArray = arraySorter.sort(arrayForSort);
+    	//then(post-conditions)
+    	int[] expectedArray = {-7612, -320, -224, -130, -112, -68, -43, -27, -27, -18, -8};
```

Comment:
expected array should stay in //given(pre-conditions) block, when and then - just one string